### ISSUE TITLE
Atomically update VSCode to avoid `code` being missing

### ISSUE THREAD

**Issue by BookOfGreg** [Microsoft/vscode#65009]
If VSCode is automatically updating itself in the background and you happen to try open a file in the command line, it says VSCode is missing:

```bash
/src$ code .
-bash: /mnt/c/Users/me/AppData/local/Programs/Microsoft VS Code/bin/code: No such file or directory
```
If you could leave the old code there and swap it with the new code once update is complete it may provide a nicer experience.

**Comment by joaomoreno** [comment]
How long does it take for Code to update in the background in your case?

**Comment by BookOfGreg** [comment]
It's not a long time, this is the first I've seen it so I haven't measured.
I'd have thought it would be a simple upgrade to make the new one at `code.new` and then just `mv code code.old && mv code.new code` at the end of the update. Obviously oversimplified example.

**Comment by joaomoreno** [comment]
I will have to close this as won't fix, it's just not worth the engineering effort as it would further complicate things during the already error-prone update phase. The command `code` comes back as soon as the update is done, right?

**Comment by BookOfGreg** [comment]
Yes it does come back immediately after update so it's only missing a short time.
Thanks for your time considering this.
